### DISCUSSION TITLE
公式识别训练

### DISCUSSION THREAD

**Discussion by songge25** [OleehyO/TexTeller#64] [Q&A]
感谢您带来这么优秀的工作，我想在这个模型上微调，我看src目录下提供了一个数据示例，请问您的端到端模型和段落识别的ocr模型，是同一个模型吗？
我想做一个段落识别的效果，不需要整页识别，请问可以按照这个示例开始吗

**Comment by OleehyO** [comment]
texteller的段落识别除了公式识别模型外，还用到了paddleocr的文字检测和文字识别模型。但是对于很小的段落，texteller本身其实也具备段落识别的能力，所以可以从所给的示例开始构建数据集进行训练。
